zap/src/main/java/org/zaproxy/zap/view/MultipleRegexesOptionsPanel.java

```diff
@@ -76,6 +83,50 @@ public MultipleRegexesOptionsPanel(Dialog owner, RegexesTableModel model) {
 
         this.owner = owner;
         getTable().setSortOrder(0, SortOrder.ASCENDING);
+        JButton importButton =
+                new JButton(Constant.messages.getString("context.include.import.label"));
+        importButton.addActionListener(e -> importButtonClicked(e));
+        addButton(importButton);
+    }
+
+    public void importButtonClicked(ActionEvent event) {
+        File file = selectImportFile();
+        if (file != null) {
+            List<String> regexes = getRegexes();
+
+            String line = null;
+            try (BufferedReader br = new BufferedReader(new FileReader(file))) {
```

Comment:
Should use a specific charset (e.g. UTF-8).

Reply:
If no charset is given, then the system default is used which should be the most appropriate one.

Reply:
On a second thought, while the system default might be the best guess for the file encoding it would still be undefined behavior. So I changed it to use UTF-8.

Reply:
FileReader with charset is not availble in java 8, but java 8 build is required by the autobuild. I could go with InputStreamReader which is available in both, Java 8 and Java 11, but I checked the zap code for other places where files are read while I found a couple of places where FileReader is used without giving a charset, there was no InputStreamReader solution, so I decided to revert the changes and at least be consistent with th other code.

Reply:
Can use NIO, e.g. `Files.newBufferedReader(file.toPath(), StandardCharsets.UTF_8)`.

Reply:
changed to NIO